### PR TITLE
Troll genderfluid changes!

### PR DESCRIPTION
**Pull request by MaidKiller**
Eu não sei
bowser

### CODE REVIEW

File: code/_globalvars/signs.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+GLOBAL_LIST_INIT(signsGold, list("Gemnius","Gemsci","Gemza"))
+GLOBAL_LIST_INIT(signsLime, list("Canrus","Cansci","Canrist"))
+GLOBAL_LIST_INIT(signsOlive, list("Lesces","Leiborn","Lerius"))
+GLOBAL_LIST_INIT(signsJade, list("Virrus","Virnius","Virpia"))
+GLOBAL_LIST_INIT(signsTeal, list("Libun","Ligo","Limino"))
+GLOBAL_LIST_INIT(signsCerulean, list("Scorittarius","Scorun","Scorza"))
+GLOBAL_LIST_INIT(signsIndigo, list("Sagio","Sagimino","Sagicen"))
+GLOBAL_LIST_INIT(signsPurple, list("Capriza","Capries","Capriborn"))
+GLOBAL_LIST_INIT(signsViolet, list("Aquapio","Aquaries","Aquapia"))
+GLOBAL_LIST_INIT(signsFuchsia, list("Pirius","Pittarius","Picen")) //Pintius kkkk
```
Comment (ChangelingRain) [inline code/_globalvars/signs.dm]: you are a coward! you have not added twelve signs per color!

**Comment by BebeYoshi** [comment]
Saia daqui brasileiro

**Comment by BebeYoshi** [comment]
macaco

**Review by BebeYoshi — CHANGES_REQUESTED** [review]
Corno é tu desgraçado

**Comment by Tlaltecuhtli** [comment]
🐒 alert

**Comment by BebeYoshi** [comment]
@SouDescolado better review this one

**Comment by nicbn** [comment]
Que porra é essa mermão

File: config/admins.txt [hunk 2]
```diff
@@ -9,133 +9,5 @@
...
-Plizzard = Game Master
-octareenroon91 = Game Master
-Serpentarium = Game Master
-Averagejoe82 = Game Master
-The Dreamweaver = Game Master
-88Naoki = Game Master
-Naksuasdf = Game Master
-MrDoomBringer = Game Master
-shizcalev = Game Master
+Zanexsas = Host
```
Comment (MMMiracles) [inline config/admins.txt]: new host coming through beep beep

**Comment by PKPenguin321** [comment]
okay, this is epic

**Comment by MMMiracles** [comment]
what makes it even funnier is that the config changes make me think this is just his local that he fucks around on

**Comment by Jordie0608** [comment]
I found a gif which illustrates my reaction to this PR
man just

**Comment by sagattyy** [comment]
why

**Comment by sagattyy** [comment]
mandioca dura

**Comment by AffectedArc07** [comment]
yes

**Pull request closed without merging**